libcxx/utils/libcxx/test/features.py

```diff
@@ -183,25 +183,26 @@ def _getAndroidDeviceApi(cfg):
         actions=[AddLinkFlag("-latomic")],
     ),
     Feature(
-        name="non-lockfree-atomics",
+        name="has-64-bit-atomics",
         when=lambda cfg: sourceBuilds(
             cfg,
             """
             #include <atomic>
-            struct Large { int storage[100]; };
+            struct Large { char storage[64/8]; };
             std::atomic<Large> x;
-            int main(int, char**) { (void)x.load(); return 0; }
+            int main(int, char**) { (void)x.load(); x.is_lockfree(); return 0; }
```

Comment:
Both these tests seem to fail in the CI right now when they should not. Is it possible that you should use `(void)x.is_lockfree();` instead? I suspect a unused-return-value warning is failing this test all the time.

Reply:
Can we validate how this test fails when it fails? 

This really scares me. It's exactly the kind of thing which could silently break.

Also, what's the underlying system property which defines if atomics are lock free? The target arch?
Can we validate that this test passes or fails appropriately for different architectures.

Reply:
More generally speaking, I think what we'd want is a test where we:

1. Test that we have the expected Lit features defined on various common platforms
2. Maybe validate that we have exactly the expected set of `UNSUPPORTED` tests in various common configurations

This would catch a mistake like this, and more. It's a bit hard to implement, though, cause we can't use the test suite for that (or it would be very brittle).